### PR TITLE
Split ooi and OpenStack providers

### PR DESCRIPTION
**Pull request by enolfc**
# Summary 

This moves the OCCI-specific code of the OpenStack provider to a new ooi provider. This will allow to easily support antive OpenStack without OCCI and to adapt to publishing to AMS using independent topics for each service endpoint in GOCDB.

### CODE REVIEW

**Comment by enolfc** [comment]
To be rebased once the other pending merges are done

**Comment by enolfc** [comment]
How are we dealing with the codeclimate issues? Should we have dedicated PRs for that?

File: README.md [hunk 1]
```diff
@@ -272,6 +274,27 @@ and test it:
...
+                            --os-user-domain-name default --os-project-name <tenant> \
+                            --os-project-domain-name default --os-auth-url <auth-url>
+cloud-info-provider-service --yaml /etc/cloud-info-provider/openstack.yaml \
+                            --middleware ooi \
+                            --os-username <username> --os-password <password> \
+                            --os-user-domain-name default --os-project-name <tenant> \
+                            --os-project-domain-name default --os-auth-url <auth-url>
+```
+
+
```
Comment (gwarf) [inline README.md]: Probably one of the empty lines is not needed

Comment (enolfc) [reply README.md]: Fixed

**Pull request merged**